src/Configurations/test/Responses.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { waitFor } from '@testing-library/react';
+
+import * as components from '@folio/stripes-acq-components';
+
+import { mockKy, rest, server } from '../../test/net';
+import { ERROR_RESPONSE, renderConfigurations, url } from './setup';
+
+
+jest.mock('@folio/stripes-acq-components', () => ({
+  ...jest.requireActual('@folio/stripes-acq-components'),
+  useShowCallout: jest.fn(),
+}));
+
+jest.mock('react-virtualized-auto-sizer', () => ({ children }) => children({ width: 1920, height: 1080 }));
+
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  useOkapiKy: () => mockKy,
+  useStripes: () => ({
+    hasPerm: jest.fn().mockReturnValue(true),
+  }),
+  IfPermission: props => <>{props.children}</>,
+}));
+
+
+describe('EditorLayer', () => {
+  it('`useSingleQuery` fires showCalloutHook if any error', async () => {
+    server.use(rest.get(url.configurations.single, ERROR_RESPONSE));
+
+    const mockShowCallout = jest.fn().mockImplementation(() => jest.fn());
+
+    jest.spyOn(components, 'useShowCallout').mockImplementation(() => mockShowCallout);
+
+    renderConfigurations('/1/edit');
```

Comment:
This is sort of fragile...
It will break if the routing implementation changes. In such case it would be preferable to align only the "Routing" section tests, but not these ones.

I'd propose to make a setup function, something like `renderEditor()` which would call `renderConfigurations()` without the path parameter, and then make all the way to the editor by finding and clicking appropriate **user controls**.
That would be [much more reliable way](https://twitter.com/kentcdodds/status/977018512689455106).